web-client/src/views/TrialSessionWorkingCopy/WorkingCopySessionList.jsx

```diff
@@ -107,14 +109,17 @@ export const WorkingCopySessionList = connect(
               <th colSpan="2">Trial Status</th>
             </tr>
           </thead>
-          {trialSessionWorkingCopyHelper.formattedCases.map(item => {
-            return (
-              <CaseListRowTrialSession
-                formattedCase={item}
-                key={item.docketNumber}
-              />
-            );
-          })}
+          <tbody>
+            {trialSessionWorkingCopyHelper.formattedCases.map(item => {
+              return (
+                <CaseListRowTrialSession
+                  formattedCase={item}
+                  key={item.docketNumber}
+                  trialSessionWorkingCopy={trialSessionWorkingCopy}
```

Comment:
Looks like this component already pulls in this helper off of state, is there a reason it needs to be explicitly passed in?

Reply:
Ah ok thanks!